src/components/Image.js

```diff
@@ -74,22 +76,24 @@ export const Image = ({
         ? CacheManager.get(sourceProp.uri)
             .getPath()
             .then((path) => {
-              mounted && setSource({ uri: path });
+              mounted && setSource({ uri: path ?? NO_IMAGE.uri });
             })
             .catch((err) => {
               console.warn(
                 'An error occurred with cache management for an image',
                 sourceProp.uri,
                 err
               );
-              mounted && setSource(sourceProp);
+              mounted && setSource(NO_IMAGE);
             })
-        : mounted && setSource(sourceProp);
+        : mounted && setSource(NO_IMAGE);
     }
 
     return () => (mounted = false);
   }, [timestamp, refreshInterval, sourceProp, setSource]);
 
+  if (source?.uri === 'NO_IMAGE') return null;
```

Comment:
we should better check for `NO_IMAGE.uri` here instead of hardcoded string, right?

Reply:
✅